packages/editor-ui/src/components/LeadEnrichment/LeadEnrichmentSection.vue

```diff
@@ -38,23 +38,42 @@ const sectionTemplates = computed(() => {
 			id: index,
 			name: workflow.title,
 			workflows: [{ id: index }],
-			nodes: workflow.nodes as ITemplatesNode[],
+			nodes: removeDuplicateNodes(workflow.nodes as ITemplatesNode[]),
 		});
 	});
 	return carouselCollections;
 });
 
+// For lead enrichment we only care about app nodes so trigger nodes are removed
+function removeDuplicateNodes(objects: ITemplatesNode[]): ITemplatesNode[] {
+	const uniqueNames = new Set<string>();
+
+	return objects.filter((node) => {
+		const plainName = node.displayName.replace(' Trigger', '');
+		if (uniqueNames.has(plainName)) {
```

Comment:
would not be better to check best on the node id `n8n-node-base.facebook` vs `n8n-node-base.facebookTrigger`?

Reply:
Talked with @RicardoE105, we will be doing this on back-end

Reply:
Talked with @RicardoE105, this logic will be moved to back-end